.drone.yml

```diff
@@ -755,8 +755,8 @@ steps:
   image: grafana/build-container:1.4.1
   commands:
   - "[ $(grep FocusConvey -R pkg | wc -l) -eq \"0\" ] || exit 1"
-  - ./bin/grabpl test-backend --edition oss
-  - ./bin/grabpl integration-tests --edition oss
+  - ./bin/grabpl test-backend --edition oss --tries 5
+  - ./bin/grabpl integration-tests --edition oss --tries 5
```

Comment:
@aknuds1 Seems awkward, but isn't it the correct way to declare the retries, since they are running in parallel?

Reply:
I'm unsure what you mean? What runs in parallel?

Reply:
They run under in the same step, so when the first finishes the second starts (scratch parallel). DYT that they could be two different steps?